app/translations/en.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
@ihsavru I think we should short this strings alphabetically so that it becomes easier to add/update translation in future.